analyzer/testdata/src/debug/contains_replacements_test.go

```diff
@@ -0,0 +1,70 @@
+package debug
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContainsNotContainsReplacements_String(t *testing.T) {
```

Comment:
I might have missed something but I would expected this here
```suggestion
func TestContainsContainsReplacements_String(t *testing.T) {
```

Reply:
No, `Contains` + `NotContains`

consistent with `TestSameNotSameReplacements`

Reply:
When you know it, you know it 😂🤣

It's obvious now.

Thanks